Plugins/NativeCamera/Source/Android/CameraWrappers.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <android/api-level.h>
+#include <dlfcn.h>
+#include <map>
+
+// The camera API's in the NDK are only supported in API24 and higher. We
+// use dlopen and dlsym to link the camera lib at runtime only if the current
+// device supports it. In order to avoid redefining the type information the
+// below macro's forcefully import the camera header files as if the min supported
+// SDK was set to 24.
+#define __ORIG_ANDROID_API__ __ANDROID_API__
+#undef __ANDROID_API__
+#define __ANDROID_API__ 24
+namespace API24 {
```

Comment:
nit:
```suggestion
namespace API24
{
```

Reply:
I think this was Android Studio auto formatting my file on changes... or I just made mistakes :)